src/test/resources/focasorin/suite-mobiledesktop.xml

```diff
@@ -0,0 +1,15 @@
+<suite name="Mobile Test Suite">
+    <test name="Mobile Test">
```

Comment:
You're missing groups tags for include mobile here that will be run, or you can exclude desktop ones.